pkg/crc/machine/bundle/metadata_test.go

```diff
@@ -289,3 +289,27 @@ func TestGetBundleInfoFromNameInvalid(t *testing.T) {
 	_, err = GetBundleInfoFromName("crc_nanoshift_libvirt_4.16.7_amd64_232.crcbundle")
 	assert.Error(t, err)
 }
+
+func TestGetBundleNameFromURI(t *testing.T) {
+	// URI with no tag
+	bundleName, err := GetBundleNameFromURI("docker://quay.io/crcont/openshift-bundle")
+	assert.Equal(t, "", bundleName)
+	assert.Error(t, err)
+
+	expectedBundleName := "crc_libvirt_4.17.3_amd64.crcbundle"
```

Comment:
This needs to be OS dependent (`switch runtime.GOOS { ... }` for example), currently this fails on macos/windows:
```
=== RUN   TestGetBundleNameFromURI
    metadata_test.go:304: 
        	Error Trace:	/Users/runner/work/crc/crc/pkg/crc/machine/bundle/metadata_test.go:304
        	Error:      	Not equal: 
        	            	expected: "crc_libvirt_4.17.3_amd64.crcbundle"
        	            	actual  : "crc_vfkit_4.17.3_amd64.crcbundle"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-crc_libvirt_4.17.3_amd64.crcbundle
        	            	+crc_vfkit_4.17.3_amd64.crcbundle
        	Test:       	TestGetBundleNameFromURI
--- FAIL: TestGetBundleNameFromURI (0.00s)
```

```
=== RUN   TestGetBundleNameFromURI
    metadata_test.go:304: 
        	Error Trace:	D:/a/crc/crc/pkg/crc/machine/bundle/metadata_test.go:304
        	Error:      	Not equal: 
        	            	expected: "crc_libvirt_4.17.3_amd64.crcbundle"
        	            	actual  : "crc_hyperv_4.17.3_amd64.crcbundle"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-crc_libvirt_4.17.3_amd64.crcbundle
        	            	+crc_hyperv_4.17.3_amd64.crcbundle
        	Test:       	TestGetBundleNameFromURI
--- FAIL: TestGetBundleNameFromURI (0.00s)
```

Reply:
I will have a look and modify the name based on the OS! Thanks for the review :+1: